repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -2871,8 +2871,28 @@ private void handleEntityMutation(List<AtlasVertex> vertices) throws AtlasBaseEx
         RequestContext.get().endMetricRecord(metricRecorder);
     }
 
-}
-
-
+    @Override
+    @GraphTransaction
+    public void moveBusinessPolicy(Set<String> policyIds, String assetId, String type) throws AtlasBaseException {
+        // Start performance metric recording
+        AtlasPerfMetrics.MetricRecorder metric = RequestContext.get().startMetricRecord("moveBusinessPolicy.GraphTransaction");
```

Comment:
This can be avoided, overall API metrics can be found in API audits loggings as well as the Perf loggings
AtlasPerfMetrics.MetricRecorder is primarily used for method-level metrics

Reply:
Please ignore this